src/Process/Pool/Logger/Message/Process/FromProcessInterface/Builder.php

```diff
@@ -0,0 +1,42 @@
+<?php
+declare(strict_types=1);
+
+namespace Neighborhoods\Kojo\Process\Pool\Logger\Message\Process\FromProcessInterface;
+
+use Neighborhoods\Kojo\Process\Pool\Logger\Message\Process\FromProcessInterfaceInterface;
+use Neighborhoods\Kojo\ProcessInterface;
+
+class Builder implements BuilderInterface
+{
+    use Factory\AwareTrait;
+    /** @var ProcessInterface */
+    protected $processInterface;
+
+    public function build() : FromProcessInterfaceInterface
+    {
+        $fromProcessInterface = $this->getProcessPoolLoggerMessageProcessFromProcessInterfaceFactory()->create();
+        $fromProcessInterface->setProcessInterface($this->getProcessInterface());
```

Comment:
I think stripping out those useful fields should happen here, and this method should return an `array`